### ISSUE TITLE
HmIP-FALMOT-C12

### ISSUE THREAD

**Issue by atze09** [danielperna84/pyhomematic#367]
**Home Assistant version (if applicable):**
2020.12.7

Hello,
I wanted to ask if it is possible to add supoort for the HmIP-FALMOT-C12.
I can support if that is possible.
Or is this in progress?

Best regards,
Dominik

**Comment by rbloehm** [comment]
@atze09 What are you trying to achieve by integrating that device?

In the standard use cases I can think of you just need the wall thermostats to be integrated into HA, and the floor heating actuators are just configured once (e.g., via CCU) to work correctly with the thermostats and valve actuators.

**Comment by leinich** [comment]
@rbloehm 
Although I only own HMIP-FAL230 devices, my usecase would be to be able to track the valve state in HA.
The difference between those two devices is that FALMOT reports 0-100%  and FAL230 true(open)/false(closed).

Would like to see some kind of Integration 😉

**Comment by rbloehm** [comment]
I see, the valve state is interesting. Thanks for the clarification.

Related discussion and starting point for solution: 
https://community.home-assistant.io/t/homematic-ip-hmip-falmot-c12-valve-level/150343/14

**Comment by atze09** [comment]
Yes valve state is the interesting thing. For further improvement this could me helpful.

**Comment by atze09** [comment]
> I see, the valve state is interesting. Thanks for the clarification.
> 
> Related discussion and starting point for solution:
> https://community.home-assistant.io/t/homematic-ip-hmip-falmot-c12-valve-level/150343/14

This is the discussion I have opened in parallel.
But how can I edit files in the root system of home assistant?

**Comment by danielperna84** [comment]
I have added the code from the forum into the version I have just released. Closing this issue for now. If further features of the device are required, a separate issue can be opened for that.

**Comment by atze09** [comment]
> I have added the code from the forum into the version I have just released. Closing this issue for now. If further features of the device are required, a separate issue can be opened for that.

Awesome thank you Daniel.
How can I update it in my home assistant docker?
Is there a tutorial to do so?

**Comment by danielperna84** [comment]
I think it will take several weeks before this gets merged into the official Home Assistant.
If you want to apply the change manually, please head to the Forum. The question has been asked and answered multiple times there. I don't use Docker, so I don't know how to do it.

**Comment by atze09** [comment]
> I think it will take several weeks before this gets merged into the official Home Assistant.
> If you want to apply the change manually, please head to the Forum. The question has been asked and answered multiple times there. I don't use Docker, so I don't know how to do it.

I didn't find something suitable but I will get it working;).
Thanks for quick reply and adding of the falmot.
And a great new year to you.

**Comment by atze09** [comment]
I did not get it working in my running system. Where can i see, when this gets merged into official home assistant?

**Comment by danielperna84** [comment]
I think the submitted update was too late to make it into the current release. Since the release-schedule of Home Assistant now is monthly, it will probably be available in the new release we'll get in february.

**Comment by thecem** [comment]
Valve Position is available now, but how we get the values into HA?
\thx a lot!

[link](https://community.home-assistant.io/t/homematic-ip-hmip-falmot-c12-valve-level/150343/23?u=thecem)

**Comment by danielperna84** [comment]
@thecem 

This project is not being worked on anymore. Switch to the new integration for better device support: https://github.com/danielperna84/custom_homematic
